tools/ou_period_statistics_retune.py

```
@@ -0,0 +1,416 @@
+    env["W3D_VALIDATION_WINDOW_SEC"] = str(WINDOW_SEC)
+    if env_name:
+        env[env_name] = value
+    if segments:
+        env["W3D_VALIDATION_SEGMENTS"] = ",".join(
+            f"{name}:{lo:.9g}:{hi:.9g}" for name, lo, hi in SEGMENTS
+        )
+    if seed is not None:
+        env["W3D_IMU_SEED"] = str(seed)
+        env["W3D_INIT_SEED"] = str(1000 + seed)
+
+    completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)